.claude/commands/automaker-admin.md

```
@@ -0,0 +1,456 @@
+
+Toutes les requêtes API nécessitent l'en-tête `X-API-Key` sauf `/api/auth/*`, `/api/setup/*`, `/api/health`.
+
+Pour récupérer la clé API :
+
+```bash
+API_KEY=$(cat /home/r2d2helm/projects/automaker/apps/server/data/.api-key 2>/dev/null)
+```
+
+### Santé
+
+```
```

markdownlint-cli2 (0.20.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)